### DISCUSSION TITLE
scikit-bio 0.6.0 released!

### DISCUSSION THREAD

**Discussion by qiyunzhu** [scikit-bio/scikit-bio#2001] [Announcements]
We are excited to announce [scikit-bio 0.6.0](https://github.com/scikit-bio/scikit-bio/releases/tag/0.6.0)!

This release introduces significant enhancements and features, including the launch of a new website, the addition of new modules, classes and algorithms, and support for the latest versions of Python and SciPy. This release also includes various optimizations, bug fixes, and documentation improvements, making it a substantial upgrade for users and developers alike.

**Release highlights:**

- Launched a new scikit-bio website (scikit.bio) with improved design and documentation.
- Added a data table module and support for BIOM tables.
- Added a sample metadata class and support for numeric and categorical columns.
- Enhanced biological sequence operations, including alphabet, substitution matrix and index conversion.
- Implemented a differential abundance test using Dirichilet multinomial distribution, which mirrors ALDEx2.
- Implemented generalized phylogenetic alpha diversity metrics.
- Adopted NumPy’s new random generator to enable reproducible analysis.
- Support for Python 3.12+ and SciPy 1.11+.
- Improvements of performance, robustness and usability in various functionality.

Review the [changelog](https://github.com/biocore/scikit-bio/blob/main/CHANGELOG.md#version-060) for a complete list of the changes. Browse the [documentation](https://scikit.bio/docs/0.6.0/index.html) to learn about what you can do with scikit-bio. Follow [@scikitbio](https://twitter.com/scikitbio) for project updates. Thanks for your interest in scikit-bio!

**Comment by ebolyen** [comment]
Congrats on the release!
